src/widgets/AutocompleteSelector.js

```diff
@@ -93,12 +93,11 @@ export class AutocompleteSelector extends React.PureComponent {
       <View style={localStyles.container}>
         <SearchBar
           autoCapitalize="none"
-          autoCorrect={false}
+          autoCorrect={true}
           autoFocus
-          color="white"
-          onChange={text => this.setState({ queryText: text })}
           placeholder={placeholderText}
-          placeholderTextColor="white"
+          debouncedCallback
```

Comment:
There is no prop `debouncedCallback` in the `SearchBar` component! Was this a mistake?

Reply:
yes, I think so. already been removed.